src/CpAmm.ts

```diff
@@ -1733,6 +1739,7 @@ export class CpAmm {
    */
   async swap(params: SwapParams): TxBuilder {
     const {
+      feePayer,
```

Comment:
Why need feePayer in swap ix?

Reply:
I think some integrators may offer gasless token swaps for users

Reply:
pls remove this, we just need that feature when claiming fee